composer.json

```diff
@@ -18,7 +22,7 @@
         }
     ],
     "require": {
-        "php": "^7.2.5|^8.0",
+        "php": "^7.2.5|^8.0|^8.1",
```

Comment:
^8.0 means that you can use it on 8.1 as well.

Reply:
The problem is not here. Problem in running assembly for php 8.1 - Run unit tests without coverage